README.md

```diff
@@ -85,3 +85,38 @@ The template comes with a set of pre-defined NPM commands (to run as `npm run <c
 
 ## Manual testing in the browser
 To test, either load as a temporary add-on in Firefox (`about:debugging`) or Chrome ("developer mode" in `chrome://extensions`) or use `npm run start`.
+
+# Publishing a study
+## Create an addons.mozilla.org account
+
+Create account on https://addons.mozilla.org, navigate to the "Developer Hub", and choose "Submit a New Add-on".
```

Comment:
@rhelmer @knowtheory did we settle on the ownership of the page? @rhelmer last time raised a good point about the security model of this process, i.e. what if the page gets hijacked after a study is approved and added to our approved list?

Reply:
One potentially centralized way to control this, for now, is to allow studies with an addon-id+version tuple, instead of just their id.

Reply:
> One potentially centralized way to control this, for now, is to allow studies with an addon-id+version tuple, instead of just their id.

Sorry I don't think I totally understand this bit... do you mean for controlling which version of a particular study we make "live" on remote settings? I think we can already do that by pointing at the specific version of an add-on by URL. 

But, it does not prevent existing study users from being upgraded, since the owner of the AMO page controls this.

Reply:
> But, it does not prevent existing study users from being upgraded, since the owner of the AMO page controls this.

Good point. We need to think carefully about this.

Reply:
@knowtheory could you please make a call on the ownership of AMO pages for studies? The implication is that we can no longer control when updates to studies happen, this would be up to partners.

Using remote-settings, we can still control when they go live and we can pause them, and they won't be able to send data until we authorize them.

I think this is ready to land otherwise.